src/api/response/iiif/manifest.js

```diff
@@ -259,6 +256,37 @@ function transform(response) {
       }
     }
 
+    /** Add logo manually (w/o IIIF Builder) */
+    const nulLogo = {
+      id: "https://iiif.dc.library.northwestern.edu/iiif/2/00000000-0000-0000-0000-000000000003/full/pct:50/0/default.webp",
+      type: "Image",
+      format: "image/webp",
+      height: 139,
+      width: 1190,
+    };
+    jsonManifest.logo = [nulLogo];
+
+    /** Add provider manually (w/o IIIF Builder) */
+    const provider = {
+      id: "https://dc.library.northwestern.edu/",
+      type: "Agent",
+      label: { en: ["Northwestern University Libraries"] },
+      homepage: [
+        {
+          id: "https://dc.library.northwestern.edu/",
+          type: "Text",
+          label: {
+            en: [
+              "Northwestern University Libraries Digital Collections Homepage",
+            ],
+          },
+          format: "text/html",
```

Comment:
Add language. In this case, this is the language of the declared URI in `id`.

```suggestion
          format: "text/html",
          language: [
            "en"
          ]
```